### ISSUE TITLE
TypeError: __repr__ returned non-string (type list) when running deforest on CF template

### ISSUE THREAD

**Issue by hawry** [hawry/deforest#3]
**Describe the bug**
Running deforest on a CF template containing lists fails during debug logging.

**Expected behavior**
Do not fail the entire run if the logging calls don't work, especially for debug level logs.

**Screenshots or error logs**
```
$ deforest -d template.yml
DEBUG: parsing file 'template.yml'
DEBUG: read 33678 bytes from file
DEBUG: received 33678 bytes of data to clean
DEBUG: enabling CF tags
DEBUG: parsing yaml
DEBUG: creating cleaner queue
DEBUG: created queue with size 0
DEBUG: created <deforest.cleaners.CloudFormationCleaner instance at 0x7f08bc323638>
DEBUG: created <deforest.cleaners.DefaultCleaner instance at 0x7f08bc32dbd8>
DEBUG: created <deforest.cleaners.IgnoreCleaner instance at 0x7f08bc32dc20>
DEBUG: <deforest.cleaners.CloudFormationCleaner instance at 0x7f08bc323638> clean
Traceback (most recent call last):
  File "/home/hawry/.local/bin/deforest", line 11, in <module>
    sys.exit(main())
  File "/home/hawry/.local/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/hawry/.local/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/hawry/.local/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/hawry/.local/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/hawry/.local/lib/python2.7/site-packages/deforest/deforest.py", line 26, in main
    cleaned = f.clean()
  File "/home/hawry/.local/lib/python2.7/site-packages/deforest/filecleaner.py", line 48, in clean
    self.cleaner_queue.get().clean()
  File "/home/hawry/.local/lib/python2.7/site-packages/deforest/cleaners.py", line 18, in clean
    logging.debug("checking {}".format(self.caller.result))
TypeError: __repr__ returned non-string (type list)
```

**Desktop (please complete the following information):**
WSL on Win 10

**OAS/Swagger version:**
OAS 3

**Additional context**
CloudFormation template version 2010-09-09

**Comment by hawry** [comment]
Should be fixed in #5 . Waiting to see if error persists before closing.

**Comment by hawry** [comment]
Fixed in 664629b1
